week3/homework/src/index.js

```diff
@@ -1,3 +1,75 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+
+const Express = require('express');
+const bodyParser = require('body-parser');
+
+const {
+  validateTask,
+  appendToDo,
+  removeToDo,
+  readToDoList,
+  clearTodos,
+  updateToDoList,
+  markTodo
+} = require('./utilities');
+
+const app = Express();
+app.use(bodyParser.json());
+
+app.use(Express.json());
+
+app.post('/todos', (req, res, next) => {
+  validateTask(req.body);
+  const { todo } = req.body;
+  appendToDo(todo)
+    .then(data => res.send(data))
+    .catch(err => next(err.message));
+});
+
+app.get('/todos', (req, res, next) => {
+  readToDoList()
+    .then(data => res.send(data))
+    .catch(err => next(err.message));
+});
+
+app.put('/todos/:id', (req, res, next) => {
+  validateTask(req.body);
+  const { id } = req.params;
+  const { todo } = req.body;
+
+  updateToDoList(id, todo)
+    .then(data => res.send(data))
+    .catch(err => next(err.message));
+});
+
+app.delete('/todos/:id', (req, res, next) => {
+  const { id } = req.params;
+
+  removeToDo(id)
+    .then(data => res.send(data))
+    .catch(err => next(err.message));
+});
+app.delete('/todos', (req, res, next) => {
```

Comment:
Make sure you give your functions some space by adding a line in between them, as you did above.